src/apiData/getData.js

```diff
@@ -1,27 +1,35 @@
-export const travelApi = () => {
-  const displayErrMessage = (err) => {
-    const message = err.message === 'Failed to fetch' ? 'Something went wrong. Please check your internet connection' : err.message;
-  }
+const displayErrMessage = (err) => {
+  const message = err.message === 'Failed to fetch' ? 'Something went wrong. Please check your internet connection' : err.message;
+};
 
-  const travelersApi = fetch('http://localhost:3001/api/v1/travelers')
-    .then(response => response.json())
-    .catch(err => displayErrMessage(err));
+const getTraveler = () => {
+  return fetch('http://localhost:3001/api/v1/travelers')
+  .then(response => response.json())
+  .catch(err => displayErrMessage(err));
+};
 
-  const tripsApi = fetch('http://localhost:3001/api/v1/trips')
-    .then(response => response.json())
-    .catch(err => displayErrMessage(err));
+const getTrips = () => {
+return fetch('http://localhost:3001/api/v1/trips')
+  .then(response => response.json())
+  .catch(err => displayErrMessage(err));
+};
 
-  const destinationApi = fetch('http://localhost:3001/api/v1/destinations')
-    .then(response => response.json())
-    .catch(err => displayErrMessage(err));
+const getDestinations = () => {
+  return fetch('http://localhost:3001/api/v1/destinations')
+  .then(response => response.json())
+  .catch(err => displayErrMessage(err));
+}
```

Comment:
Could look into adding a function that checks for an error to stop any 400 level errors
Probably wont be a problem on a fetch though